savePositionData.js

```diff
@@ -54,7 +100,66 @@ async function savePositionDataSQL(
   });
 }
 
+const retriveInitalAndHistoricalData = async (position) => {
+  // load position from mint txHash
+  const tx = await queryTheGraphForMintTransactHash(position);
+  console.log(position);
+  const initData = await loadPositionInitDataByTxHash(tx, position);
+  if (!initData) return [null, null];
+
+  const historicalData = await fetchHistoricalPriceData(
+    initData.token0symbol,
+    initData.token1symbol,
+    initData.blockTimestemp
+  );
+  if (!historicalData) {
+    logger.error("No historical data found for position", position.id);
+    notify(
+      "🪙🕰️ Action Needed 🕰️🪙",
+      `No historical data found for position ${position.id} for tokens ${initData.token0symbol} and ${initData.token1symbol} plase do it maually`
```

Comment:
add chain name
**please do it manually**

Reply:
Done